src/pages/home/report/ReportActionItemBasicMessage.tsx

```diff
@@ -12,7 +13,7 @@ function ReportActionItemBasicMessage({message, children}: ReportActionItemBasic
     const styles = useThemeStyles();
     return (
         <View>
-            <Text style={[styles.chatItemMessage, styles.colorMuted]}>{message}</Text>
+            <Text style={[styles.chatItemMessage, styles.colorMuted]}>{Str.htmlDecode(message)}</Text>
```

Comment:
Maybe could memoize the decoded message but it probably doesn't make a significant difference